src/brsTypes/components/RoSGNode.ts

```diff
@@ -1442,10 +1456,12 @@ function addFields(interpreter: Interpreter, node: RoSGNode, typeDef: ComponentD
             }
 
             // Add the onChange callback if it exists.
-            let observeField = node.getMethod("observeField");
```

Comment:
So this block was wrong, wasn't it? We weren't supposed to call the callback when adding fileds

Reply:
I don't think we were ever actually calling the callback when adding those fields, but we were running through `Callable#call`, which has its own quirks with `m`.  I switched to manually implementing the small amount of `observeField` that we need here, since the logic is pretty simple 😄